src/go/rdctl/cmd/shell.go

```diff
@@ -99,3 +110,57 @@ func setupLimaHome() error {
 	os.Setenv("LIMA_HOME", candidatePath)
 	return nil
 }
+
+const restartDirective = "Either run 'rdctl start' or start the Rancher Desktop application first"
+
+func checkLimaIsRunning(commandName string) bool {
+	output, err := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}").CombinedOutput()
```

Comment:
Same as Windows — do we need `CombinedOutput`? Or is just stdout enough?
(Of course, this has the side effect of hiding the error, so maybe that's what we want? If yes, please add code comments.)

Reply:
Both these commands are specific enough that if there actually is error output then things are running very badly.